docs/examples/example-cascade-temperature-control.md

```diff
@@ -0,0 +1,64 @@
=5.0;}\nif(temp>30) {temp=30.0;}\n//Many ovens cannot deal with decimals.\ntemp = Math.round(temp)\nmsg.heater_setpoint=temp\nreturn msg","outputs":1,"noerr":0,"initialize":"","finalize":"","libs":[],"x":590,"y":140,"wires":[["45925567a1177d5b"]]},{"id":"5fa07ec0c1842823","type":"api-current-state","z":"29fde6c61057362b","name":"Current climate settings","server":"e2dd69fb.8f70a8","version":3,"outputs":1,"halt_if":"","halt_if_type":"str","halt_if_compare":"is","entity_id":"climate.my_climate","state_type":"str","blockInputOverrides":false,"outputProperties":[{"property":"payload","propertyType":"msg","value":"","valueType":"entityState"},{"property":"data","propertyType":"msg","value":"","valueType":"entity"}],"for":0,"forType":"num","forUnits":"minutes","x":950,"y":140,"wires":[["ba7e48582e04c14c","0a7c09c92a01178e"]]},{"id":"ba7e48582e04c14c","type":"switch","z":"29fde6c61057362b","name":"Not same?","property":"data.attributes.temperature","propertyType":"msg","rules":[{"t":"neq","v":"heater_setpoint","vt":"msg"}],"checkall":"true","repair":false,"outputs":1,"x":1150,"y":140,"wires":[["1114bb7e63d16e98","0a7c09c92a01178e"]]},{"id":"1114bb7e63d16e98","type":"api-call-service","z":"29fde6c61057362b","name":"Sett temperatur","server":"e2dd69fb.8f70a8","version":3,"debugenabled":false,"service_domain":"climate","service":"set_temperature","entityId":"climate.my_climate","data":"{\"entity_id\":\"climate.my_climate\",\"temperature\":\"{{heater_setpoint}}\"}","dataType":"json","mergecontext":"","mustacheAltTags":false,"outputProperties":[],"queue":"none","x":1320,"y":140,"wires":[[]]},{"id":"45925567a1177d5b","type":"api-current-state","z":"29fde6c61057362b","name":"Enabled?","server":"e2dd69fb.8f70a8","version":3,"outputs":2,"halt_if":"true","halt_if_type":"bool","halt_if_compare":"is","entity_id":"input_boolean.cascade_enabled","state_type":"habool","blockInputOverrides":false,"outputProperties":[{"property":"enabled","propertyType":"msg","value":"","valueType":"entityState"},{"property":"data","propertyType":"msg","value":"","valueType":"entity"}],"for":0,"forType":"num","forUnits":"minutes","x":760,"y":140,"wires":[["5fa07ec0c1842823"],[]]},{"id":"0a7c09c92a01178e","type":"debug","z":"29fde6c61057362b","name":"","active":false,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":1330,"y":240,"wires":[]},{"id":"ee1904546f36bdcb","type":"delay","z":"29fde6c61057362b","name":"","pauseType":"delay","timeout":"100","timeoutUnits":"milliseconds","rate":"1","nbRateUnits":"1","rateUnits":"second","randomFirst":"1","randomLast":"5","randomUnits":"seconds","drop":false,"allowrate":false,"outputs":1,"x":290,"y":160,"wires":[["6f77dd60c21180e1"]]},{"id":"e2dd69fb.8f70a8","type":"server","name":"Home Assistant","version":2,"addon":true,"rejectUnauthorizedCerts":true,"ha_boolean":"y|yes|true|on|home|open","connectionDelay":false,"cacheJson":true,"heartbeat":true,"heartbeatInterval":"30"}]
```

Comment:
The flows looks better pretty printed. Since they are very long, it could be a good idea either to wrap them in a details-container or have them in separate files.

A details container is made like this:

```
::: details Title
content here, including code block
:::
```

Reply:
Done